src/gdcmodels/init_index.py

```diff
@@ -119,11 +126,20 @@ def init_index(args: Arguments):
     es_models = gdcmodels.get_es_models(vestigial_included=False)
     es = get_elasticsearch(args)
 
-    for index in args.index:
+    aliases = ["" for _ in args.index]
```

Comment:
Can you comment on this construct? It is initializing a list with empty strings to match the indices. That smells like a map or named tuples. Beware keeping two lists of synced items.